### ISSUE TITLE
 SELECT COUNT(*) Results In CLI Shutting Down

### ISSUE THREAD

**Issue by earlev4** [duckdb/duckdb#5087]
### What happens?

Hi. Thank you so much for all the work on DuckDB! Truly an amazing project. 

Working through some exercises on example database (world.db), specifically the `Country` table and COUNT(*) results in DuckDB quitting.

Any guidance would be greatly appreciate. 

Thank you!

### To Reproduce

Running DuckDB CLI (v0.5.1) on Windows 11. 

From the command line:  
1. duckdb
2. `INSTALL sqlite_scanner; LOAD sqlite_scanner;`
3. `CALL sqlite_attach('world.db');`
4. `SELECT COUNT(*) FROM Country;`

Results in duckdb closing and returning to the command prompt.

However, `SELECT COUNT(Region) FROM Country;` returns the proper results.

| count("Region") |
|-----------------|
| 239             |

Also, installed DuckDB Pre Python (v0.5.2.), performed the same steps, and resulted in:  

`Kernel Restarting
The kernel for duckdbtest.ipynb appears to have died. It will restart automatically.`

### OS:

Windows 11

### DuckDB Version:

0.5.1

### DuckDB Client:

CLI

### Full Name:

Earle Lyons

### Affiliation:

Regis University

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by earlev4** [comment]
Performing a COUNT on each individual column returns the expected results with no issues. Thank you.

**Comment by Alex-Monahan** [comment]
Hello! Sorry you are running into this. Could you attach the example database or provide a link to it? It helps to be able to fully reproduce the issue.

**Comment by earlev4** [comment]
Hi @Alex-Monahan! Thanks so much for the fast response! Attached is the example database. I appreciate the help. Thanks again! 
[world.zip](https://github.com/duckdb/duckdb/files/9865163/world.zip)

**Comment by Tishj** [comment]
This is probably an issue of the sqlite_scanner, and ideally this issue should be raised there
But I'll investigate regardless

**Comment by Tishj** [comment]
```
D select count(Region) FROM Country;
┌─────────────────┐
│ count("Region") │
│      int64      │
├─────────────────┤
│             239 │
└─────────────────┘
D select count(*) from country;
┌──────────────┐
│ count_star() │
│    int64     │
├──────────────┤
│          239 │
└──────────────┘
```
This is on MacOS, so I fear the issue is related to Windows, which is a pain to debug

**Comment by ededovic** [comment]
I have a similar issue in 0.6.0 version on both Win 10 and Linux. It crashes the Python console if I run this query
```
select count(*) from mysql_databases
Process finished with exit code -1073741819 (0xC0000005)
```

I can not upload the database, but the table structure is pretty simple and there is not that much data in it: 
```
-- auto-generated definition
create table mysql_databases
(
    db     varchar(255),
    server varchar(255)
);
```

**Comment by Mytherin** [comment]
Thanks for the report! I have pushed a fix in duckdblabs/sqlite_scanner#20. Please file issues related to extensions in their corresponding repositories in the future.

**Comment by earlev4** [comment]
Thanks so much for the fix @Mytherin! I sincerely appreciate the work on it. My apologies for creating the issue under the wrong repository.
